src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -63,10 +69,15 @@ synchronized void restart(boolean started) {
 		this.totalBlocks = total;
 		this.minBlocks = min;
 		this.fetchedBlocks = fetched;
+        // clone() because Date is mutable
+		this.latestSuccess
```

Comment:
The whitespace here is inconsistent - some lines are indented with tabs and others with spaces. You are free to keep it as-is if you like.